practices/responsibility.md

```diff
@@ -0,0 +1,91 @@
+---
+slug: responsibility
+stages:
+  - development
+short_description: Responsibility is keeping up the trust and faith of people who trusts you, no matter what the situation is. Responsibility primarily involves two things -- authority and accountability; however, trust is often a factor as well.
+
+tags:
+  - responsibility
+  - team work
+  - ability
+  - response
+  - appreciation
+  - problems
+  - problem-solving
+  - assign
+  - leader
+
+keywords:
+  - responsibility
+  - trust
+  - people
+  - benefits
+---
+
+# Responsibility
+
+**TL;DR**
+
+Responsibility is keeping up the trust and faith of people who trusts you, no matter what the situation is.
+
+
+## What Is Responsibility
+Responsibility is the combination of two words: **Response + ability**. It also means taking on more risk and therefore reward because it involves a degree of success and failure beyond oneself. People are responsible for themselves and so when failure happens, you are just failing yourself. When you choose to take on responsibilities beyond yourself, you are agreeing to take the interests of others into your hands in trying to do more, be more, and allow others to benefit from your being responsible.
+
+Responsibility is not for everyone. But it’s a powerful and deeply rewarding ship to navigate once embarked.
+
+![Responsibility](https://shiftyourfamilybusiness.com/wp-content/uploads/2018/05/ResponsibilityBlog-845x321.jpg)
+
+
+## Why You Might Want Responsibility
+
+Being responsible brings us many benefits. It can help us achieve our goals and objectives in any area of our life. Responsibility allows us to create principles, morals and helps us to lead your life. Being a responsible person helps us to:
+
+**Be more independent:** Assuming the consequences of our actions will help us make better decisions.
+**Be more reliable**: By being responsible, we gain other's trust and we will also gain confidence in ourselves
+**Be more honest:** When we tend to tell the truth and keep our promises, the people around us will believe us and see us as an honest person.
+
+
+
+## Problems Responsibility Solves
+
+- Establishes team accountability
+- Key to great team work
+- Quality of output is usually high as the tasks done by the team are mostly interdependent
+- An individual is committed to complete the tasks allocated to them
+- Problem solving is given importance rather than blaming others
+- Makes your team members feel important and appreciated
+- Helps an individual do more, faster
+
+
+## How to Implement Responsibility
+
+- **Start by Talking**: The first step is to talk to the individuals concerned
+- **Ensure Adequate Resources**: A sensible action is to ensure that your people have the resources they need to do their job . This might include providing training, equipment, access to mentors  and coaches , and suchlike.
```

Comment:
I am sorry, our plagiarism check found the same text in a mindtools article. The mentioned text is: _A sensible early action is to ensure that your people have the resources they need to do their job  . This might include providing training, equipment, access to mentors and coaches, and suchlike._ Could you please rewrite it or rephrase it?